### DISCUSSION TITLE
Custom domains for Observable Cloud data apps

### DISCUSSION THREAD

**Discussion by jamesking** [observablehq/framework#1675] [Q&A] [answered]
I would like to use Observable Framework to publish a site. 

I would like to use Observable Cloud for this, but I want a custom domain. Is this in the roadmap in the near future?

**Comment by ananya-roy** [comment, marked as answer]
Hi James  👋
This feature is available today in our [Enterprise tier](https://observablehq.com/pricing). Please reach out at support@observablehq.com and we can talk through the process of enabling this for your workspace.

Thanks!

**Reply by jamesking** [reply]
Thanks @ananya-roy
